programs/management/commands/tests/test_import_all_program_configs.py

```
@@ -0,0 +1,214 @@
+        self.assertEqual(record.white_label_code, "co")
+        self.assertIsNotNone(record.imported_at)
+
+    def test_model_unique_filename(self):
+        """Test that filename must be unique"""
+        ProgramConfigImport.objects.create(
+            filename="test_program.json",
+            program_name="test_program",
+            white_label_code="co",
+        )
+
+        with self.assertRaises(Exception):
```

Ruff (0.14.14)

Do not assert blind exception: `Exception`

(B017)